lib/deepl.rb

```diff
@@ -38,6 +40,11 @@ def api
     @api ||= API.new(configuration)
   end
 
+  def glossary(name, entries, source_lang, target_lang, options = {})
```

Comment:
I think that adding all glossary methods to the API interface as a whole might be quite dirty. `api.create_glossary`, `api.glossary`, `api.list_glossaries`, `api.glossary_entries`, ...

What about if a new chain method is added? Something like `api.glossaries.create`, `api.glossaries.find`, `api.glossaries.list`, `api.glossaries.entries`, ...

That `api.glossaries` might return a new `Api::GlossaryApi` wrapper.